provisioning/playbook.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Install the latest git package
+  # hosts: localhost
+  hosts: all
+  connection: local
+  user: root
+  become: yes
+
+
+  tasks:
+  - name:  Install Python3 package
+    package:  
+      name=python3
```

Comment:
It's better to use the convention:

name: 'python3'
state: 'latest'

Reply:
Thanks a lot for feedback !
Anyway i rollback this PR because someone already did this task.